support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -128,6 +138,37 @@ function sendEventWhenReadyTrigger(sendEventWhenReady: () => void): void {
 	}
 }
 
+function sendEventAcquisitionDataFromQueryParamEvent(
+	acquisitionData: ReferrerAcquisitionData,
+): void {
+	void ifQmPermitted(() => {
```

Comment:
I don't think you need to wrap this code in `ifQmPermitted` as this conditional check already happens inside the `init` function.